src_tests/test_exact_distribution_calculator.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from numpy import array, std
+
+from src.ExactLossyBosonSamplingDistributionCalculator.ExactLossyBosonSamplingDistributionCalculator \
```

Comment:
I believe that there is one folder too many. It would be worth it to put everything into the folder if there were more classes that actually share interface, purpose or logic. In this case, I'd say it's overkill.

Reply:
Initially I wanted to separate dataclass and calculator class -> thus another folder was created. Later I realized that I'm not sure if dataclass will be used in other places, to I decided to put it together with calculator in one file. You're right that in that case it's kind of overkill. I'll move the file out of the folder and remove the folder.

Reply:
+1